src/pages/projectSettings/tabs/NotificationsTab/getFormSchema.tsx

```diff
@@ -58,6 +63,53 @@ export const getFormSchema = (
             },
           },
         },
+        ...(projectType !== ProjectType.Repo && {
+          banner: {
```

Comment:
this is a small thing but organization-wise I think the project banner section might belong better above the subscription section (the subscription section could be very long, so you'd have to scroll all the way to the end to access the project banner section)